### ISSUE TITLE
Settings Not Saving

### ISSUE THREAD

**Issue by SpaceCore12090** [joelshepherd/tabliss#284]
I am a Google Chrome user, and whenever I close Google Chrome and reopen it my settings reset to default.

**Comment by CraftyDH** [comment]
@SpaceCore12090 this is potentially a duplicate of #268.

**Comment by Hanif1601** [comment]
Absolutely. I report that bug not so long ago.

**Comment by josemando** [comment]
This is also happening to me using Firefox 83.0 on macOS Catalina

**Comment by joelshepherd** [comment]
Yep, the release went out yesterday. If you are experiencing issues from version 2.2.0, email me your setup at support@tabliss.io and I can help troubleshoot your issue.
